### PR TITLE
Add LICENSE file

### PR DESCRIPTION
**Pull request by hardysim**
Adding a `LICENSE`-file so one can see it at a glance in GitHub.

image

### CODE REVIEW

**Comment by hardysim** [comment]
@dvdciri can you have a look?

You're referencing the Apache 2.0 License in `README.md` but there's no `LICENSE` -file.

**Comment by dvdciri** [comment]
You're right.
Don't need to reference the whole license, just add whatever is on the readme inside the LICENSE file and change the readme to `Apache 2.0 © Davide Cirillo`.

Thanks

**Comment by hardysim** [comment]
> Don't need to reference the whole license

I'm not sure how GitHub works here. The intention is, that the license is recognized correctly. So not including *the whole* license seems not to work.

I've created a [demo-project](https://github.com/hardysim/test) with a `LICENSE` containing only the stuff from your `README.md` and it doesn't show up in the header (should look like in the screenshot).

---

I think a good way to include the copyright notice is to keep that in the `README.md`  and let `LICENSE` only contain the original license.

So basically just what you've said but with the hole license:

> and change the readme to `Apache 2.0 © Davide Cirillo`

---

@dvdciri Please acknowledge and I'll add the change for `README.md`.

**Comment by dvdciri** [comment]
@hardysim Very sorry for the late reply but i didn't really have time to look at this project in the past months. Please move forward with the changes to the license and this will be merged
Thanks

**Comment by hardysim** [comment]
@dvdciri OK, I've updated `README.md` so everything should be fine now and ready to be merged.

**Comment by hardysim** [comment]
Hmm, there was a conflict in the README so I just copied the whole content of the file (from `master`) and just changed the licence-part but it now shows up some more changes.

Maybe there are more differences between `develop` and `master`. Please have a look.

**Comment by dvdciri** [comment]
I've updated develop from master, there were some changes. Can you please rebase your change onto develop and squash everything into 1 commit please? I'll merge that straight after

**Comment by hardysim** [comment]
OK, fixed the new conflict.

Can you use "[squash and merge](https://blog.github.com/2016-04-01-squash-your-commits/#whats-changing)" right in GitHub? I'm just changing files online right now and cannot squash them myself without cloning the whole repo. Letting GitHub squash this would be far easier.
